src/Optimizely/DecisionService/DecisionService.php

```diff
@@ -148,6 +150,152 @@ public function getVariation(Experiment $experiment, $userId, $attributes = null
   }
 
   /**
+   * Get the variation the user is bucketed into for the given FeatureFlag
+   * @param  FeatureFlag $featureFlag The feature flag the user wants to access
+   * @param  string      $userId      user id
+   * @param  array       $attributes  user attributes
+   * @return array/null  {"experiment" : Experiment, "variation": Variation } / null
+   */
+  public function getVariationForFeature(FeatureFlag $featureFlag, $userId, $attributes){
```

Comment:
Please change it to userAttributes or filteredAttributes.

Reply:
Done